watson/watson.py

```diff
@@ -344,6 +343,7 @@ def _get_request_info(self, route):
         return dest, headers
 
     def _get_remote_projects(self):
+        import requests
```

Comment:
Can you add a comment explaining why we only import request when required?

Reply:
Comment added. Feel free to modify it to any statement you prefer.

Reply:
I did a forced push though the change does not reflect in the discussion page. Please see https://github.com/TailorDev/Watson/pull/312/files instead.